src/backend/distributed/transaction/backend_data.c

```diff
@@ -351,6 +357,42 @@ get_all_active_transactions(PG_FUNCTION_ARGS)
 }
 
 
+/*
+ * pg_cancel_backend overrides the Postgres' pg_cancel_backend to cancel
+ * a query with a global pid
+ */
+Datum
+pg_cancel_backend(PG_FUNCTION_ARGS)
+{
+	CheckCitusVersion(ERROR);
+
+	uint64 pid = PG_GETARG_INT64(0);
+
+	bool withTerminate = false;
+	bool success = EndGlobalProcess(pid, withTerminate);
+
+	PG_RETURN_BOOL(success);
+}
+
+
+/*
+ * pg_terminate_backend overrides the Postgres' pg_terminate_backend to terminate
+ * a query with a global pid
+ */
+Datum
+pg_terminate_backend(PG_FUNCTION_ARGS)
```

Comment:
Postgres' `pg_terminate_backend` has `timeout` as well: 
```SQL
│ pg_catalog │ pg_terminate_backend │ boolean          │ pid integer, timeout bigint DEFAULT 0 │ func │
```

I think we should have the exact same signature and behaviour